wp-lazy-loading.php

```diff
@@ -134,33 +133,17 @@ function wp_lazy_loading_enabled( $tag_name, $context ) {
  * @return string Converted content with 'loading' attributes added to images.
  */
 function wp_add_lazy_load_attributes( $content, $context = null ) {
-	$tags = array();
-
-	if ( null === $context ) {
-		$context = current_filter();
-	}
-
-	if ( wp_lazy_loading_enabled( 'img', $context ) ) {
-		$tags[] = 'img';
-	}
-
-	// Experimental. Will be removed when merging unless the HTML specs are updated by that time.
-	if ( wp_lazy_loading_enabled( 'iframe', $context ) ) {
-		$tags[] = 'iframe';
-	}
-
-	if ( empty( $tags ) ) {
+	if ( ! wp_lazy_loading_enabled( 'img', $context ) ) {
 		return $content;
 	}
 
 	return preg_replace_callback(
-		'/<(' . implode( '|', $tags ) . ')(\s)[^>]+>/',
+		'/<img(\s)[^>]+>/',
 		function( array $matches ) {
 			if ( ! preg_match( '/\sloading\s*=/', $matches[0] ) ) {
```

Comment:
The regex here should look for `\b` not `\s` at the beginning. Same considerations as above, if the img tag was formatted with each attribute starting on a new line, chances are that `loading` won't have a white-space char before it.